modules/Emails/EmailsController.php

```diff
@@ -449,6 +450,24 @@ public function action_GetFromFields()
             }
         }
 
+        if ($sugar_config['email_allow_send_as_user']) {
```

Comment:
can you please check if this email setting is set like: `isset($sugar_config['email_allow_send_as_user'])`

Reply:
I have the config option set as a bool. If it's set to false, isset will return true. What is the best practice in this case? Should it be unset instead of set to false? I personally like it being explicit, and if it is unset, this still returns false. However, if the project prefers it unset if disabled, I'll change the code. I think perhaps wrapping it in an isset is the way to go. As written it will produce an undefined variable warning if the email settings haven't been saved after the patch. Please let  me know.